apps/throttles.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework.throttling import SimpleRateThrottle
+from influxdb_metrics.loader import log_metric
+
+
+class OrgApiLimit(SimpleRateThrottle):
+    """
+    API rate limiter for req/day
+    """
+    scope = 'org_api_limit'
+    rate = '1000/day'  # default rate
```

Comment:
Why'd the rate get moved into the throttle class? I think if we need a default rate, I'd rather have it defined in the settings file. But I'm also unsure of this rate - I think the default rate should be unlimited if not set in the JWT.

Reply:
Ah, I was thinking I had to move it into the model to be able to update it later, but that's not the case. I'll remove it and then set the default rate in the settings.